funding_requests.md

```diff
@@ -0,0 +1,27 @@
+---
+### Document pending approval
+## Summary
+In this document we outline the procedure and protocol of handling funding requests. 
+This document serves as a guideline and reference point for both Funding Requests applicants and the community and council that has to judge and approve these requests.
+Funding Requests can only be granted upon discussion and an official vote to approve the request in a Grin Community Council meeting (biweekly meetings in Grin #General on Keybase).
+
+## Funding
+Funding request can be made in any currency. However, for the ease of discussing we prefer requests are made in major currencies like euros or dollars.
+We advise to make a funding request in the following format:
+
+- a) Total amount of funding to be paid upon completion of the project
+- b) Total amount of funding, sub payments to be made upon completion of predefined milestones
+- c) Any other funding model if the applicant can provide good arguments to deviate from option a) and b)
+
+## Payments
+Payments will be made after reviewing the completed project or completed project milestones.
+Payments will by default be made 85% in BTC and 20% (15% base + 5% bonus) in Grin. 
+This effectively means applicants will be paid 105% of the amount they requested if they applicant choses to be paid in 15% or more in Grin.
+Payment in Grin will be based on a monthly average value while Bitcoin  
+In case an applicant prefers to be paid less than the default 15% +5% bonus payment in Grin, we expect sound argumentation why this deviation is required, such as personal circumstances.
```

Comment:
Probably add that the +5% bonus is dependent on accepting >=15% payment in grin.